### ISSUE TITLE
mdl-tooltip not working

### ISSUE THREAD

**Issue by gtibbitts** [developit/preact#456]
I am trying to convert an app from react to preact. I can't get tooltips to work. When I hover over the element tagged with a mdl-tooltip class, it is supposed to append the is-active class. It is also supposed to insert a style attribute to the span. Neither of these occur with preact. This appears to be a different issue than the span issue. It appears mdl can't find the elements with the mdl-tooltip class or can't apply the styles. One thing that is happening is before the page finishes loading, even after ready is called, mdl can't find the elements. If I add the call to the componentDidMount it finds them, but still doesn't add the styles to render the tooltip. I am using preact 7.1.0. I've tried this with mdl and preact-mdl with the same result.

**Comment by developit** [comment]
Hi there! Can you try with Preact 6.4? There are some changes in Preact 7.x that might cause issues in preact-mdl which have not yet been accounted for here.

**Comment by gtibbitts** [comment]
I should have mentioned that I've tried 6.4 also. 7.1 seems to work the best at this point. In 6.4 some other issues exist that work in 7.1. This issue is the same in 6.4 and 7.1.

**Comment by gtibbitts** [comment]
I should also mention that this is happening inside a Griddle component cell.

**Comment by developit** [comment]
Never heard of griddle...

**Comment by gtibbitts** [comment]
It's a grid component. It uses tables, so the mdl-tooltip is rendered inside tr/td tags. I'm not sure if Griddle is what is causing the problem. The same code works fine in React, but it could be something between Griddle and preact causing the issue.

**Comment by developit** [comment]
Ok. Hard to specifically address without a repro, but I can address the removal issue and we'll see if that helps.

**Comment by gtibbitts** [comment]
I'll see what I can do on a repro. Griddle may be reloading a large section of the dom and so the removal issue may solve it.

**Comment by developit** [comment]
Yup yup, I'm thinking they could be related in a weird way like that. The removalists is nice because it's not specific to MDL at all.

**Comment by marvinhagemeister** [comment]
A lot has changed in Preact during the past years. We just released an alpha for our 10th version a few weeks ago which is a complete rewrite of Preact. It brings a lot of compatibility fixes and works much better with third-party libraries.

Unfortunately there is no repro or [codesandbox](https://codesandbox.io) attached to this issue so I can't verify if it has been fixed. Because this issue is quite old I'm assuming that it is resolved. Feel free to ping me to re-open it if that'S not the case :+1:
